core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -153,8 +153,8 @@
          aria-expanded="false"
          aria-label="User Menu"
          tabindex="0"
-         ng-mouseover="$ctrl.openSubmenu($event, 'profileMenu')"
-         ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)">
+         ng-mouseover="$ctrl.openSubmenu($event, 'profileMenu')">
+         <!--ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)">-->
```

Comment:
Why this change?

Reply:
This is in reference to issue #11542
When the mouse pointer leaves, it stops the menu from being closed and hence retains the background color of the profile tab. This fixed the error.

Reply:
Okay, then please remove this line instead of commenting it.